lib/grpc/logger/client.ex

```diff
@@ -13,13 +11,19 @@ defmodule GRPC.Logger.Client do
       {:ok, channel} = GRPC.Stub.connect("localhost:50051", interceptors: [{GRPC.Logger.Client, level: :info}])
   """
 
+  require Logger
+
+  @behaviour GRPC.ClientInterceptor
+
+  @impl true
   def init(opts) do
-    level = Keyword.get(opts, :level) || :info
+    level = opts[:level] || :info
```

Comment:
```suggestion
    level = Keyword.get(opts, :level) || :info
```

I don't want to support maps here